### PR TITLE
[chore] Updates to scrape duration

### PR DESCRIPTION
**Pull request by MovieStoreGuy**
**Description:** 

- This is to prepare for the future release of core that includes scraping from collector start.
- Fixes accidental change to apache receiver's default collection interval.

**Link to tracking Issue:**

https://github.com/open-telemetry/opentelemetry-collector/pull/7635

**Testing:**
NA

**Documentation:**
NA

### CODE REVIEW

File: .chloggen/msg_revert-default-apache-receiver.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: reverted default collection time back to 10s from 1m
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [23030]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```
Comment (astencel-sumo) [inline .chloggen/msg_revert-default-apache-receiver.yaml:20]: Perhaps useful to add that the change was mistakenly added in v0.78.0?

**Comment by MovieStoreGuy** [comment]
Don't merge til https://github.com/open-telemetry/opentelemetry-collector/pull/7635 is merged.

**Comment by astencel-sumo** [comment]
I think this PR also needs to update the docs of all the affected receivers to add the new `initial_delay` configuration option.

**Comment by astencel-sumo** [comment]
This is great. Now I wonder: is this a breaking change? :thinking: It changes the time of the first scrape for all the affected components - from the value of `collection_interval` (which is `10s` or `60s` for most components) to `1s`. Is this change OK? Or should we rather make the default `initial_delay` be the same as `collection_interval` so that there's no change in (the default) behavior? :thinking: Perhaps it's fine, I cannot think of an issue that might arise from this change.

**Review by djaglowski — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Comment by MovieStoreGuy** [comment]
> This is great. Now I wonder: is this a breaking change? 🤔 It changes the time of the first scrape for all the affected components - from the value of `collection_interval` (which is `10s` or `60s` for most components) to `1s`. Is this change OK? Or should we rather make the default `initial_delay` be the same as `collection_interval` so that there's no change in (the default) behavior? 🤔 Perhaps it's fine, I cannot think of an issue that might arise from this change.

The interval changes would only really impact users who have larger intervals since their current expectations is to wait that to capture data. The gut feeling is that collection intervals under 1m don't notice since something like 10s over a duration of an hour is hidden is most observability tools rollups or aggregations.

**Pull request merged**